server/engine/src/main/java/org/eclipse/lsp/cobol/service/CobolTextDocumentService.java

```diff
@@ -286,8 +285,9 @@ private RelatedFullDocumentDiagnosticReport getRelatedFullDocumentDiagnosticRepo
                                     forLeft(new FullDocumentDiagnosticReport(entry.getValue()))))
                 .flatMap(map -> map.entrySet().stream())
                 .collect(toMap(Entry::getKey, Entry::getValue));
-    relatedFullDocumentDiagnosticReport.setRelatedDocuments(relatedDocDiagnostics);
-    return relatedFullDocumentDiagnosticReport;
+    RelatedUnchangedDocumentDiagnosticReport relatedUnchangedDocumentDiagnosticReport = new RelatedUnchangedDocumentDiagnosticReport();
+    relatedUnchangedDocumentDiagnosticReport.setRelatedDocuments(relatedDocDiagnostics);
+    return relatedUnchangedDocumentDiagnosticReport;
```

Comment:
Our diagnostics are always unchanged as we wait for the analysis to finish and are not sending any incremental diagnostics.